latex_access/tests/test_preprocessor.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+import tempfile
+import pickle
+try:
+    from latex_access import preprocessor
+except ModuleNotFoundError:
```

Comment:
`ModuleNotFoundError` was added in Python 3. Since these tests should be backwards compatible please use `ImportError` here.